streams/src/test/java/org/apache/kafka/streams/integration/KStreamAggregationIntegrationTest.java

```diff
@@ -1,1150 +0,0 @@
-        consumerProperties.setProperty(ConsumerConfig.GROUP_ID_CONFIG, "group-" + safeTestName);
-        consumerProperties.setProperty(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "earliest");
-        consumerProperties.setProperty(ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG, keyDeserializer.getClass().getName());
-        consumerProperties.setProperty(ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG, valueDeserializer.getClass().getName());
-        consumerProperties.put(StreamsConfig.WINDOW_SIZE_MS_CONFIG, 500L);
-        if (keyDeserializer instanceof TimeWindowedDeserializer || keyDeserializer instanceof SessionWindowedDeserializer) {
-            consumerProperties.setProperty(StreamsConfig.WINDOWED_INNER_CLASS_SERDE,
-                Serdes.serdeFrom(innerClass).getClass().getName());
-        }
-        return IntegrationTestUtils.waitUntilMinKeyValueWithTimestampRecordsReceived(
-            consumerProperties,
-            outputTopic,
-            numMessages,
-            60 * 1000);
-    }
-
-    private <K, V> String readWindowedKeyedMessagesViaConsoleConsumer(final Deserializer<K> keyDeserializer,
-                                                                      final Deserializer<V> valueDeserializer,
-                                                                      final Class innerClass,
-                                                                      final int numMessages,
-                                                                      final boolean printTimestamp) throws Exception {
-        final ByteArrayOutputStream newConsole = new ByteArrayOutputStream();
-        final PrintStream originalStream = System.out;
-        try (final PrintStream newStream = new PrintStream(newConsole)) {
-            System.setOut(newStream);
-
-            final String keySeparator = ", ";
-            // manually construct the console consumer argument array
-            final String[] args = new String[] {
-                "--bootstrap-server", CLUSTER.bootstrapServers(),
-                "--from-beginning",
-                "--property", "print.key=true",
-                "--property", "print.timestamp=" + printTimestamp,
-                "--topic", outputTopic,
-                "--max-messages", String.valueOf(numMessages),
-                "--property", "key.deserializer=" + keyDeserializer.getClass().getName(),
-                "--property", "value.deserializer=" + valueDeserializer.getClass().getName(),
-                "--property", "key.separator=" + keySeparator,
-                "--property", "key.deserializer." + StreamsConfig.WINDOWED_INNER_CLASS_SERDE + "=" + Serdes.serdeFrom(innerClass).getClass().getName(),
-                "--property", "key.deserializer.window.size.ms=500",
-            };
-
-            ConsoleConsumer.run(new ConsoleConsumerOptions(args));
-            newStream.flush();
-            System.setOut(originalStream);
-            return newConsole.toString();
-        }
-    }
-}
```

Comment:
Why delete this test? I don't have an issue, just wanting to understand.  I have the same comment for the other deleted integration tests.